src/components/FooterPage/Footer.scss

```diff
@@ -0,0 +1,41 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0px;
```

Comment:
usually this would be in the app.scss , fortunately you don't need this portion for this code.  Please remove this.

Reply:
I removed it. Thanks